controllers/drplacementcontrol_controller_test.go

```diff
@@ -681,7 +674,7 @@ var _ = Describe("DRPlacementControl Reconciler", func() {
 
 				updateClonedPlacementRuleStatus(userPlacementRule, drpc, WestManagedCluster)
 				// Force the reconciler to execute by changing one of the drpc.Spec fields. We chose s3Endpoint
-				setDRPCSpecExpectationTo(drpc, "newS3Endpoint", rmn.ActionFailover, "")
+				setDRPCSpecExpectationTo(drpc, rmn.ActionFailover, "")
```

Comment:
comment in 676 says the reason I added that parameter (that you removed) is to force the reconciler to run.  Without it, that test case will not run.  I may just remove the NOOP test cases.  They are causing more headaches at the moment.

Reply:
The NOOP is a good test case and I prefer it stays, especially given that you have coded it already. Perhaps we could add a reconcileRequest label, with current time stamp as the value, to the DRPC object to force the reconciler to run?